index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e746991a9e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <div class="logo">
+            <p class="top">9:45</p>
+            <div class="ico">
+                <i class="fa-solid fa-signal fa"></i>
+                <i class="fa-solid fa-wifi fa"></i>
+                <i class="fa-light fa-battery-full fa"></i>
+            </div>
+            <h3 class="lo">IBUTE~UCHENNA</h3>
+        </div>
+        <div class="menu">
+            <img src="img/v1.png" class="img1 img">
+            <img src="img/v1.png" class="img2 img">
+            <img src="img/v1.png" class="img3 img">
+        </div>
```

Comment:
- [ ] I see that you are stacking three images above each other to get the Hamburger menu icon. I suggest you to use [font awesome](https://fontawesome.com/) `<i class="fa-solid fa-bars"></i>` as you are already using it in some other parts of your project 🔎

- [ ] Please use a `nav` tag to wrap your Hamburger button instead of `div`. This way you are making sure you follow the semantic rules and helping search engines and other user devices to determine the importance and context of your web page 💁🏽‍♂️.